# Brief introduction  
The genomic mechanisms that enable or limit the evolution of major changes in organismal body plans remain one of the great questions of evolutionary biology. Living members of Mollusca represent **the greatest morphological disparity** of any animal phylum: squid, worms, living rocks and candy-coloured tree snails, as well as intermediate and additional novel forms known from the fossil record. Early work on chromosome numbers in meitotic division used molluscs, and interest in understanding patterns in chromosome numbers across species also led to insights into animal polyploidy and the importance of genome duplication in mollusc evolution (@Hallinan & Lindberg 2011). Modern genome analyses are hindered by intrinsic factors that confound sequencing and assembly in molluscs (@REF), and so this important phylum lags in terms of quality and taxonomic coverage of whole genome sequence data. Reconstructing the evolution of genome architecture through deep time in diverse molluscs remains critical to understand the fundamental mechanisms of genome evolution in animals and the relationship between genome and phenome.  
  
# MLGs
**Mollusca Linkage groups (N=20)** constructed by the representative species from Conchifera and Aculifera, which enables researchers to investigate the chromosomal event in species with the chromosome-level genome.  
  
In this version, seven genomes were employed in constructing the protein database via the guideline in *Science Advances* (Simakov et al., 2022), including 1 snail (*Gibbula magus*), 1 bivalve (*Mizuhopecten yessonensis*), and 5 chitons (*Acanthochitona rubrolineata*, *Acanthochitona discrepans*, *Callochiton septemvalvis*, *Deshayesiella sirenkoi*, *Liolophura japonica*). Specifically, only proteins were kept in the database if they were present at all seven species and also the best hit from each other. It consists of **4,729** homologs (Mollusca_7sp/Mollusca_7sp.pep), with pseudo-locations (only for visualization, not the real gene order in the ancestor, shown in Mollusca_7sp/Mollusca_7sp.bed) of them via the AGORA v3.1 (Muffato et al., 2023). The file Mollusca_7sp/MLGs.color.txt shows the color scheme used in the published work.

**Guideline**:  
1) The identification of **multi-way mutual best hit (MBH)** clusters of encoding proteins among organisms or genomes is the core part. At first, the reciprocal blast hits between every two genomes were inferred by diamond blastp v2.1.8.162 (Buchfink et al., 2021) with the threshold of ‘-evalue 0.001 -max_target_seqs 50000’. Then, python was engaged in constructing the linkages of the orthologue among genomes (up to six, scripts shown in folder commands) . Of which, only the strict one-to-one linkage was selected, meaning such an orthologue was presented in all genomes. For example, three proteins were shown as protein a in organism A, protein b in organism B, and protein c in organism C. The reciprocal blast hits revealed that (1) protein a and protein b, (2) protein b and protein c, (3) protein a and protein c, so a linkage of proteins a-b-c was defined.  
2) The **linkage groups** between every two genomes were checked by Fisher’s exact test using R package macrosyntR (Sami El & Richard, 2023) with a significant threshold of p-value below 0.001, but it might define the group as an insignificant item falsely if the number of linkages in the group was small. Therefore, the manual check of linkage groups between two genomes is needed using pairwise dot plots in JCVI version 1.2.7 (Tang et al., 2008), especially for organisms with genome duplication (e.g., *Acthatina fulica* and *Arion vulgaris*) or the recent insertion event (*Pomacea canaliculata*) or the two organisms with far relationships (e.g., Mollusca and Porifera). Coupling with the chromosomal location of proteins from .gff3 file, any potential linkage groups at the chromosome level would be constructed using ‘groups_six_species.py’ (a specified script). Accordingly, the ancient linkage groups (ALGs) were extracted based on the significant linkage groups on every two genomes. For example, similar to the definition of MBH, there were three chromosomal associations among three organisms: 1) chromosome 1 in organism A and chromosome 2 in organism B, 2) chromosome 1 in organism A and chromosome 3 in organism C, 3) chromosome 2 in organism B and chromosome 3 in organism C; then, it should be a linkage group from organism A-B-C as chromosome 1-2-3.  
  
# How to use
Generally, it is used to look into the chromosomal evolution in organisms. With the MLGs, it will help us to identify the ancestral stat and chromosomal events in Mollusca, including 1) the identification of chromosome duplication (or fission), insertion, and fusion and 2) the real sense of whole genome duplication.   
1) **Similarity search**: diamond blastp (suggested, or blastp in blast+) could be adopted to find the hits between them (MLGs as the query and proteins from genomes as databases). The protein was defined as a significant match against the mollusk ancestors if it was hit by at least 6 of 7 proteins from a linkage in MLGs (evalue < 0.001).  
2) **Fisher’s exact test**: The linkage groups between MLGs and the specific genome are checked by Fisher’s exact test using R package macrosyntR (Sami El & Richard, 2023) with a significant threshold of p-value below 0.001.  
3) **Visualization**: It needs your effort via Python or R. Personally, I would recommend Oxford-plot (https://bitbucket.org/viemet/public/src/master/CLG/).  
Here is an example figure, shown below.  There is a event of **whole genome duplication (WGD)** in *Athatina fulica*. The evolution from the mollusk ancestor might be via the early fusions (MLG09+MLG10+MLG14, MLG18+MLG20), the duplication (17 -> 34), and latter fusions (34 -> 31).  
![image](https://github.com/ylify/MLGs/blob/main/examples/figures/Mollusca_7sp.Acthatina_fulica.oxford_plot.jpg)


# Citation  
If you adopt the database in research, please cite the below work. Thanks!  

# Reference  
> **Buchfink**, B., Reuter, K., & Drost, H.-G. (2021). Sensitive protein alignments at tree-of-life scale using DIAMOND. Nature Methods, 18(4), 366-368. doi:10.1038/s41592-021-01101-x  
> **Muffato**, M., Louis, A., Nguyen, N. T. T., Lucas, J., Berthelot, C., & Roest Crollius, H. (2023). Reconstruction of hundreds of reference ancestral genomes across the eukaryotic kingdom. Nature Ecology & Evolution, 7(3), 355-366. doi:10.1038/s41559-022-01956-z  
> **Sami El**, H., & Richard, R. C. (2023). macrosyntR : Drawing automatically ordered Oxford Grids from standard genomic files in R. bioRxiv, 2023.2001.2026.525673. doi:10.1101/2023.01.26.525673  
> **Simakov**, O., Bredeson, J., Berkoff, K., Marletaz, F., Mitros, T., Schultz, D. T., . . . Rokhsar, D. S. (2022). Deeply conserved synteny and the evolution of metazoan chromosomes. Science Advances, 8(5), eabi5884. doi:doi:10.1126/sciadv.abi5884  
> **Tang**, H., Bowers, J. E., Wang, X., Ming, R., Alam, M., & Paterson, A. H. (2008). Synteny and Collinearity in Plant Genomes. Science, 320(5875), 486-488. doi:doi:10.1126/science.1153917  
